spec/acceptance/cassandra_spec.rb

```diff
@@ -26,16 +28,16 @@
 
     describe service('cassandra') do
       it "check Cassandra-#{version} is running and enabled" do
-        is_expected.to be_running
-        is_expected.to be_enabled
+        expect(subject).to be_running
```

Comment:
Why is this better? We use `is_expected` pretty much everywhere and if Rubocop enforces this new style, I'd be in favor of disabling that cop (unless there's a very good reason).